src/createServer.js

```diff
@@ -1,11 +1,18 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const cors = require('cors');
+const usersRouter = require('./router/users.js');
+const expensesRouter = require('./router/expenses.js');
+
+const app = express();
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  app.use(cors());
+  app.use('/users', express.json(), usersRouter);
+  app.use('/expenses', express.json(), expensesRouter);
```

Comment:
https://github.com/mate-academy/node_accounting-app-v2/pull/360#discussion_r1314244836
not fixed